commands/audit/audit.go

```diff
@@ -172,6 +172,12 @@ func RunAudit(auditParams *AuditParams) (results *xrayutils.Results, err error)
 		errGroup.Go(utils.DownloadAnalyzerManagerIfNeeded)
 	}
 
+	if auditParams.xrayGraphScanParams.XscGitInfoContext != nil {
+		if err = xrayutils.SendXscGitInfoRequestIfEnabled(auditParams.xrayGraphScanParams, xrayManager); err != nil {
+			return nil, err
+		}
+	}
```

Comment:
I would suggest to add to `xrayutils.Results` attribute for multiScanId, also maybe to add it to the `SimpleJsonResults` so that it will be avaliable at the result to the user.
That will also allow you to add tests for checking if it was set / created (not empty)

Reply:
I added it to `xrayutils.Results`. Regarding `SimpleJsonResults` I'm not sure it's a significant value to present to the user. @hadarshjfrog What do you think?

Reply:
I think it's a good idea to add it there.
Will be very beneficial for e2e tests - we just need to make sure it doesn't break any schema to customers that rely on the results of simpleJson.